python/chronos/src/bigdl/chronos/forecaster/__init__.py

```diff
@@ -64,16 +65,17 @@ def __exit__(self, *args, **opts):
     warnings.warn("Please install `bigdl-orca` to use full collection of forecasters.")
 
 # import forecasters
+PREFIX_PATH = 'bigdl.chronos.forecaster'
 if torch_available:
-    from .lstm_forecaster import LSTMForecaster
-    from .tcn_forecaster import TCNForecaster
-    from .seq2seq_forecaster import Seq2SeqForecaster
-    from .nbeats_forecaster import NBeatsForecaster
+    LSTMForecaster = LazyImport('..LSTMForecaster', pkg=PREFIX_PATH+'.lstm_forecaster')
```

Comment:
I think these 2 dots is used to avoid error in `LazyImport`'s `__call__`, I think we can further enhance the implementation of `LazyImport` rather than use some walkaroud when we call it.

Reply:
In the previous answer, the pkg parameter is also preserved in `LazyImport`, but will not be used.